etcdserver/etcdserverpb/rpc.proto

```diff
@@ -909,6 +909,8 @@ message StatusResponse {
   uint64 raftTerm = 6;
   // raftAppliedIndex is the current raft applied index of the responding member.
   uint64 raftAppliedIndex = 7;
+  // errors contains alarm/health information and status
```

Comment:
add a dot `.` at the end of the sentence.

Reply:
done